lib/fathead/python_requests/parse.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin env python
+# -*- coding: utf-8 -*-
+from bs4 import BeautifulSoup
+from glob import glob
+import re
+from urllib.parse import urljoin
+
+
+def build_article(h2_parent, page_url):
+    '''Builds fathead article entry.
+
+    Accepts h2_parent and extracts title, url, abstract and code snippets.
+    Returns a list
+    '''
+    h2 = h2_parent.find('h2')
+    title = h2.text.replace('¶', '')
+    fragment = h2_parent.find('a').get('href')
+    url = urljoin(page_url, fragment)
+    abstract = ''
+    next_sibling = h2.find_next_sibling(text=None)
+    while next_sibling:
+        if next_sibling.name == 'p':
+            next_sibling_text = next_sibling.text.replace('\n', ' ')
+            abstract += ' {}'.format(next_sibling_text)
+        elif next_sibling.name == 'div':
+            pre = next_sibling.find('pre')
+            if pre:
+                pre_text = pre.text
+                if title.startswith('Raw Response'):
+                    last_span = pre.findAll('span')[-1]
+                    original_span_text = last_span.text
+                    escaped_span_text = last_span.text
+                    escaped_span_text = bytes(escaped_span_text,
```

Comment:
After wrapping text in <p> tags and testing it on codio, identation also seems to be fine. Do let me know if the problem still persists in beta.

![untitled](https://cloud.githubusercontent.com/assets/13175534/19316256/6d461862-90a9-11e6-8e43-4b9fb94ade22.png)